src/main/java/tests/uiTests/ComPageValidationTests.java

```diff
@@ -22,6 +23,33 @@ public void comPageTitleShouldAppear(){
         homePage.clickCommitteesLink();
         System.out.println(committeesPage.isCommitteesPageAgricultureLinkDisplayed());
         driver.close();
+    }
+
+    @Test
+    public void comPageShouldAppearWithFirefox(){
+        System.setProperty("webdriver.gecko.driver", "resources/geckodriver 2");
+        FirefoxDriver driver = new FirefoxDriver();
+        HomePage homePage = new PageFactory().initElements(driver, HomePage.class);
+        CommitteesPage committeesPage = new PageFactory().initElements(driver, CommitteesPage.class);
 
+        driver.get("https://www.house.gov");
+        homePage.isHomePageZipCodeInputPresent();
+        homePage.clickCommitteesLink();
+        System.out.println(committeesPage.isCommitteesPageAgricultureLinkDisplayed());
+        driver.close();
+    }
+
+    @Test
+    public void comPageShouldWorkHeadlessTest(){
+        System.setProperty("webdriver.chrome.driver", "resources/chrome-driver");
+        WebDriver driver = new HtmlUnitDriver();
```

Comment:
So html unit driver is using chrome-driver right?
please rename your chrome-driver file as resources/chromedriver4mac

I cannot see chrome options

make  tests with chrome options too?

Reply:
Cannot get your point... There are already tests runs with chrome... Can you please elaborate that?